src/Schema/Traits/SortTrait.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace SilverStripe\GraphQL\Schema\Traits;
```

Comment:
```suggestion
namespace SilverStripe\GraphQL\Schema\Plugin;
```
Please move this into the `Plugin/` directory - we don't need a whole directory for traits.

Reply:
Since both sort plugins located in different directories `SilverStripe\GraphQL\Schema\DataObject\Plugin\QuerySort` and `SilverStripe\GraphQL\Schema\Plugin\SortPlugin`, probably it would be better to leave trait in common directory for now, while we have not decided about future architecture of our plugins.

Reply:
Ahh yup good call. Didn't notice there were two plugin directories.